packages/datagateway-common/src/views/selectionAlert.component.tsx

```diff
@@ -0,0 +1,168 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+//Note: By default auto fill to the avaiable space
+type SelectionAlertProps = {
+  width?: string;
+  marginSide?: string;
+};
+
+const selectionAlertStyles = makeStyles<Theme, SelectionAlertProps>(
+  (theme: Theme) =>
+    createStyles({
+      root: {
+        backgroundColor: 'orange',
+        color: 'white',
+        width: ({ width }) => (width === undefined ? 'auto' : width),
+        marginTop: '8px',
+        marginLeft: ({ marginSide }) =>
+          marginSide === undefined ? '0px' : marginSide,
+        marginRight: ({ marginSide }) =>
+          marginSide === undefined ? '0px' : marginSide,
+        paddingTop: '2px',
+        paddingBottom: '2px',
+      },
+      animate: {
+        backgroundColor: 'orange',
+        color: 'white',
+        width: ({ width }) => (width === undefined ? 'auto' : `${width}`),
+        marginTop: '8px',
+        marginLeft: ({ marginSide }) =>
+          marginSide === undefined ? '0px' : marginSide,
+        marginRight: ({ marginSide }) =>
+          marginSide === undefined ? '0px' : marginSide,
+        paddingTop: '2px',
+        paddingBottom: '2px',
+        animation: `$pulsate 700ms ${theme.transitions.easing.easeInOut}`,
+      },
+      '@keyframes pulsate': {
+        '0%': {
+          backgroundColor: 'orange',
+        },
+        '25%': {
+          backgroundColor: 'red',
+        },
+        '50%': {
+          backgroundColor: 'orange',
+        },
+        '75%': {
+          backgroundColor: 'red',
+        },
+        '100%': {
+          backgroundColor: 'orange',
+        },
+      },
+      text: {
+        textAlign: 'center',
+        justifyItems: 'center',
+        alignItems: 'center',
+        flex: '1',
+      },
+    })
+);
+
+const SelectionAlert = React.memo(
+  (props: {
+    selectedItems: DownloadCartItem[];
+    navigateToSelections: () => void;
+    width?: string;
+    marginSide?: string;
+  }): React.ReactElement => {
+    const classes = selectionAlertStyles({
+      width: props.width,
+      marginSide: props.marginSide,
+    });
+    const [t] = useTranslation();
+    const [alertOpen, setAlertOpen] = React.useState(false);
+    const [animating, setAnimating] = React.useState(false);
+    const [alertText, setAlertText] = React.useState('');
+    //Store number of selections on last update of this component (for keeping track of changes to cart)
+    const [numSelectedItems, setNumSelectedItems] = React.useState(0);
+
+    //Current number of selections
+    const newNumSelecItems = props.selectedItems.length;
+
+    //Check for a change and assign text based on increase or decrease
+    if (newNumSelecItems !== numSelectedItems) {
+      const difference = newNumSelecItems - numSelectedItems;
+
+      //Obtain plural if needed
+      const itemText =
+        Math.abs(difference) > 1
+          ? t('selec_alert.items')
+          : t('selec_alert.item');
```

Comment:
This is why we have an internationalisation library ;) see https://www.i18next.com/translation-function/plurals